src/dev/impl/DevToys/ViewModels/Tools/Text/StringUtilities/StringUtilitiesToolViewModel.cs

```diff
@@ -626,6 +634,182 @@ private void ExecuteInverseCaseCommand()
 
         #endregion
 
+        #region AlphabetizeCommand
+
+        public IRelayCommand AlphabetizeCommand { get; }
+
+        private void ExecuteAlphabetizeCommand()
+        {
+            string? text = _textBackup;
+            if (text is null)
```

Comment:
Please use `string.IsNullOrEmpty(text)` instead, since `"" != null`.

Reply:
Other commands only check `is null`, though?

Reply:
Good point! Can't remember much why 😅
I will close this comment for now.

Reply:
To add on to what @celluj34 said, I only checked against `null` for these reasons.

1. The existing commands only check against null.
2. Checking with `string.IsNullOrEmpty` is apparently not enough to tell the compiler that `text` is not null, forcing me to use a nasty null suppressor on subsequent lines.
   ![image](https://github.com/veler/DevToys/assets/7417301/d69df671-25fa-4767-99fd-a6ad3a2a772c)
3. The sorting logic works fine with empty strings.